webpack.config.js

```diff
@@ -133,6 +133,7 @@ const webpackConfig = {
 						'|acorn-jsx' +
 						'|d3-array' +
 						'|debug' +
+						'|marked' +
```

Comment:
What does this do?

Reply:
This tells webpack to run the `marked` package js files through babel so that they get transfromed. This needs to happen because `marked` was using destructuring in its require statements, and IE11 doesn't support destructuring.